test/util.test.js

```diff
@@ -27,3 +28,11 @@ tap.test('MERGE_METHOD should be correct for valid input', async t => {
   })
   t.equal(getInputs().MERGE_METHOD, 'merge')
 })
+
+tap.test('getPackageName should get package name from branch', async t => {
```

Comment:
should test also for inputs that does not fit happy output
should test one case at time